src/app.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import Header from './header';
+
+import JudgingRoundDisplay from './judging_round_display';
+import JudgingRoundSelector from './judging_round_selector';
+
+
+class App extends React.Component {
+    constructor(props) {
+	super(props);
+	this.state = {
```

Comment:
Like my previous comments, since we have `transform-class-properties` [on](https://github.com/masschallenge/application-allocator/blob/development/.babelrc#L3) this could be:

```js
class App extends React.Component {
  state = { judging_round: null }
```

Reply:
Addressed.